# upstream: yes

From ddbd243638c7bc2baecf43a78aff46cdc12e9b2e Mon Sep 17 00:00:00 2001
From: Peter Wu <peter@lekensteyn.nl>
Date: Tue, 26 May 2020 21:43:35 +0200
Subject: [PATCH 2/2] Add missing proc_fs.h header

Required for 'struct proc_ops' since v5.7-rc1 with commit df23e2be3d24
("acpi: Remove header dependency").
---
 bbswitch.c | 1 +
 1 file changed, 1 insertion(+)

diff --git bbswitch.c bbswitch.c
index 4ce9aae..c0c6713 100644
--- a/bbswitch.c
+++ b/bbswitch.c
@@ -35,6 +35,7 @@
 #include <linux/suspend.h>
 #include <linux/seq_file.h>
 #include <linux/pm_runtime.h>
+#include <linux/proc_fs.h>
 #include <linux/version.h>
 
 #define BBSWITCH_VERSION "0.8"
-- 
2.28.0

